src/components/DetailsInfo/detailsInfo.util.js

```diff
@@ -249,7 +249,7 @@ export const generateAlertsDetailsInfo = selectedItem => {
     AlertsDetailsInfo.triggerCriteriaDetailsInfo = triggerCriteriaContent.map(trigger => {
       return (
         <li className="details-item" key={trigger.id}>
-          <div className="details-item__header">{trigger.label}:</div>
+          <div className="details-item__header_long">{trigger.label}:</div>
```

Comment:
Have you tested these changes?
I see that the general class `details-item__header` was deleted. The common style won't be applied.